frontend/src/features/projects/common/components/ProjectNotes.tsx

```diff
@@ -17,8 +18,16 @@ export interface IProjectNotesProps {
   tooltip?: string;
   /** whether or not this text box can be interaced with */
   disabled?: boolean;
+  /** comment below label */
+  comment?: string;
 }
 
+const Comment = styled.div`
+  font-size: 12px;
+  margin-top: -100px;
+  max-width: 16.7%;
+`;
```

Comment:
I know it's a bit hacky, it was just the simplest way to have the text appear right under the label

Reply:
Does this work alright if you resize the screen?

Reply:
yep, just tried changing my browser in all directions